src/components/Select/components/SelectList/SelectList.tsx

```diff
@@ -21,8 +22,11 @@ type SelectListProps = {
     flattenOptions: FlattenOption[];
     multiple?: boolean;
     virtualized?: boolean;
+    loading?: boolean;
 };
 
+const loadingOption = {value: 'LOADING', disabled: true};
```

Comment:
`LOADING` can conflict with user values, need more specific, Symbols are perfect for this

Reply:
In case of using `Symbol` we should extend `option.value` type. Is this modification necessary?

Reply:
No, let's just use more unique string for this